shared/p2p/service.go

```diff
@@ -216,6 +217,8 @@ func (s *Server) Start() {
 		}
 
 		peersToWatch = append(peersToWatch, s.bootstrapNode, s.relayNodeAddr)
+
+		log.Info("Listening on ", s.host.Addrs())
```

Comment:
have you tried logging this out? Will `log.Info` work?

Reply:
```suggestion
		log.Infof("Listening on %s", s.host.Addrs())
```

Reply:
Yes, I did. It printed out the two ip addresses (local and router)